### PR TITLE
Added VGG16 and VGG19 model for CIFAR10 Dataset

### PR DESCRIPTION
**Pull request by avik-pal**

### CODE REVIEW

**Comment by avik-pal** [comment]
#30

File: cifar10/vgg19.jl [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+conv5_2 = Conv((3,3), 512=>512, relu, pad=(1,1), stride=(1,1))
+conv5_3 = Conv((3,3), 512=>512, relu, pad=(1,1), stride=(1,1))
+conv5_4 = Conv((3,3), 512=>512, relu, pad=(1,1), stride=(1,1))
+
+# Building the model
+
+m = Chain(
+	conv1_1,
+    conv1_2,
+    x -> maxpool(x, (2,2)),
```
Comment (CarloLucibello) [inline cifar10/vgg19.jl]: what about 
```julia
Chain( 
    Conv((3,3), 3=>64, relu, pad=(1,1), stride=(1,1)),
    Conv((3,3), 64=>64, relu, pad=(1,1), stride=(1,1)),
.....
```
to save some space?

Comment (avik-pal) [reply cifar10/vgg19.jl]: Made the changes that have been suggested

**Comment by MikeInnes** [comment]
This is great. I've just tried to simplify it a bit; let me know if it's important to keep the batchnorm models separate, but it seems reasonable to just have a sensible default here.

Looks like this depends on a Metalhead release, and the CIFAR10 download is not working there for me, so we'll get that figured out first.

**Comment by avik-pal** [comment]
Keeping one model is fine. At that time I was not being able to train the `BatchNorm` ones due to speed constraints but now it should be fine.
As for the `CIFAR10` I am looking into that

**Comment by MikeInnes** [comment]
Note the small change to `getarray`; you want `Images.channelview` to avoid writing that by hand.

Otherwise this is great, thanks a lot @avik-pal!

**Pull request merged**